src/onboarding/templates/democracy/index.js

```diff
@@ -70,10 +71,11 @@ const template = {
     {
       screen: 'voting-defaults',
       validate: ({ support, minQuorum, voteDuration }) => {
-        if (support < 1 || support > 100) {
+        // Mimic contract validation
+        if (minQuorum < 0 || minQuorum > support) {
           return false
         }
-        if (minQuorum < 0 || minQuorum > 100) {
+        if (support < 1 || support >= 100) {
```

Comment:
Wasn't sure if I should also change the filtering to only allow `99` as the max value in these fields. I think the best UX would be to provide some explainer text (similar to the multisig for duplicate addresses) that explains why 100% is impossible (due to strict thresholds).

I was also debating if we should just hide the strict thresholds, and subtract 1 from the value we ultimately get, but I think that might get more confusing given that we made the change to strict thresholds as it's more familiar with traditional voting systems.

Reply:
Why don't these fields accept decimals? I think they should, to be able to set values like `0.0...1` and `99.9...9`. For instance, right now it's impossible to define a Democracy with 200 voters where just one positive vote is enough.
I agree that doing the minus 1 trick behind the scenes may be confusing.

Reply:
I agree too about the minus 1 trick, the easiest solution for now would be to add some explanation in the text below.

Then we need:

- Helping tooltips on the labels, which were planned in the original design (tooltips will be coming soon [after Popover is available in Aragon UI](https://github.com/aragon/aragon-ui/pull/225)).
- Error states (soon in Aragon UI), to let the user type something and indicate why it doesn’t work, rather than filtering the value directly.

> Why don't these fields accept decimals?

I don’t think anything prevents us to accept decimals now, it would be nice yes 👍